.github/workflows/release.yml

```
@@ -22,14 +22,121 @@ jobs:
           echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
           nix-shell --pure --run "echo" ./shell.nix
 
-      - name: Remove prerelease label from dependencies
+      - name: Set Image Org
+        run: |
+          [ -z "${{ secrets.IMAGE_ORG }}" ] && IMAGE_ORG=openebs || IMAGE_ORG=${{ secrets.IMAGE_ORG }}
+          echo "IMAGE_ORG=${IMAGE_ORG}" >> $GITHUB_ENV
+
+      - name: Set TAG from github ref name
         run: |
-          nix-shell --pure --run "./scripts/helm/update-version-upon-release.sh" ./shell.nix
+          TAG="${{ github.ref_name }}"
+          echo "TAG=${TAG}" >> $GITHUB_ENV
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Mirror images from dev to Docker Hub
+        run: |
+          nix-shell --pure --run "./scripts/staging/mirror-images.sh \
+            --source-namespace ghcr.io/${{ env.IMAGE_ORG }}/dev \
+            --target-namespace docker.io/${{ secrets.DOCKERHUB_USERNAME }} \
```

Shouldn't it be this?

```suggestion
            --target-namespace docker.io/${{ secrets.IMAGE_ORG }} \
```

---

I am planning on removing the IMAGE_ORG thing altogether and use the `github.repository_owner` so it will be that now.